FantasyGymnastics/core/views.py

```diff
@@ -44,6 +44,14 @@ def form_valid(self, form):
         league.save()
         return redirect('view_league', pk=league.pk)
 
+class LeagueSearchResultsView(ListView):
+    model = League
+    template_name = 'core/league_search_results.html'
+
+    def get_queryset(self):
+        query = self.request.GET.get('query')
+        return League.objects.filter(name__icontains=query)
```

Comment:
can you return this as context. like we did in other generic views. and name it 'leagues' or 'searched_leagues' or whatever you think makes sense